src/ambianic/pipeline/avsource/picam.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from io import BytesIO
+import time
+from PIL import Image
+
+log = logging.getLogger(__name__)
+
+picamera_override = None
+
+class Picamera():
+
+    def __init__(self, image_format='jpeg'):
+
+        self.error = None
+        self.format = image_format
+
+        if picamera_override is None:
+
+            try:
+                import picamera
+                self.camera = picamera.PiCamera()
+            except ImportError as err:
+                log.warn("Failed to import picamera module: %s" % err)
+                self.error = err
+                return
+            except Exception as err:
+                log.warn("Error importing picamera module: %s" % err)
+                self.error = err
+                return
+        else:
+            self.camera = picamera_override.PiCamera()
+
+        self.stream = BytesIO()
+        self.camera.led = True
+        self.camera.start_preview()
```

Comment:
I don't think start_preview() is applicable for us. It asks the picam library to display camera feed to the local device display, which in our case doesn't usually exist.

Reply:
good point, thanks